api/src/main/java/org/jenkinsci/plugins/workflow/graph/FlowGraphWalker.java

```diff
@@ -13,8 +15,7 @@
  * @author Kohsuke Kawaguchi
  */
 // TODO guarantee DFS order (is it not already in DFS order?)
-// TODO implement Iterable<FlowNode>
-public class FlowGraphWalker {
+public class FlowGraphWalker implements Iterable<FlowNode> {
     // queue of nodes to visit.
     // it's a stack and not queue to visit nodes in DFS
     Stack<FlowNode> q = new Stack<FlowNode>();
```

Comment:
BTW confusion between two comment lines and name of variable. I guess at some point it used to be a `Queue<FlowNode>`.